CollabApp/CollabApp.mvc/Views/Post/Posts.cshtml

```diff
@@ -32,7 +32,21 @@
             <!-- Add more filter options here as needed -->
             <button type="submit" class="btn-btn-primary">Apply Filters</button>
         </form>
-    </div>    
+    </div>
+
+    <div class="sort-sidebar">
+        <h2>Sorting</h2>
+        <form class="filter-form" method="post" asp-controller="Post" asp-action="SortPosts">
+            <label for="sortBy">Sort by:</label>
+            <select name="sortBy" id="sortBy" class="form-control">
+                <option value="desc-comments">Most Popular First</option>
+                <option value="asc-comments">Least Popular First</option>
+                <option value="desc-date">Newest First</option>
+                <option value="asc-date">Oldest First</option>
+            </select>
+            <button type="submit" class="btn btn-primary">Sort</button>
```

Comment:
Probably, there is a typo. 'class="btn btn-primary"' should be 'class="btn-btn-primary".